mars_troughs/accumulation_model.py

```diff
@@ -16,56 +16,171 @@ class AccumulationModel(Model):
+            np.ndarray of the same size as time input containing values of 
+            accumulation rates A, in m/year
+        
+        """
+        return self.intercept + self.slope * self._ins_data_spline(time)
+    
+    def get_yt(self, time: np.ndarray):
+        """
+        Calculates the vertical distance y (in m) traveled by a point
+        in the center of the high side of the trough. This distance  is a 
+        function of the accumulation rate A as y(t)=int(A(ins(t)), dt) or 
+        dy/dt=A(ins(t))
+
+        Args:
+            time (np.ndarray): times at which we want to calculate y, in years.
+        Output:
+            np.ndarray of the same size as time input containing values of 
+            the vertical distance y, in meters.
+        
+        """
+        return  -1 *(self.slope * (self._int_ins_data_spline(time) 
+                                 - self._int_ins_data_spline(0)))
+ 
+class QuadraticInsolationAccumulation(InsolationAccumulationModel):
+    """
+    Accumulation rate A (in m/year) as a  quadratic polynomial of insolation.
+    A(ins(t)) = intercept + linearCoeff*ins(t)+ quadCoeff*ins(t)^2.
+    A is in m/year.
+
+    Args:
+        times (np.ndarray): times at which the solar insolation is known, in
+                            years.
+        insolations (np.ndarray): value of the solar insolations (in W/m^2)
+        intercept (float, optional): default is 1 m/year
+        linearCoeff (float, optional): default is 1e-6 m/year per unit
+            of solar insolation (m^3/(year*W)).
+        quadCoeff (float, optional): default is 1e-6 m/year per unit
+            of solar insolation squared (m^5/(year*W^2)).
+    """
+    def __init__(self, times, insolation, intercept: float = 1.0, 
+                 linearCoeff: float = 1e-6, quadCoeff: float = 1e-6):
+        
+        super().__init__(times, insolation)
+        
+        self.intercept=intercept
+        self.linearCoeff=linearCoeff
+        self.quadCoeff=quadCoeff
+    
+    @property
+    def parameter_names(self) -> Dict[str, float]:
+        return ["intercept", "linearCoeff", "quadCoeff"]
+    
+    def get_accumulation_at_t(self, time: np.ndarray):
+        """
+        Calculates the accumulation rate A at times "time".
+
+        Args:
+            time (np.ndarray): times at which we want to calculate A, in years.
+        Output:
+            np.ndarray of the same size as time input containing values of 
+            accumulation rates A, in m/year
+        
+        """
+        return self.intercept + self.linearCoeff * self._ins_data_spline(time) \
+                              + self.quadCoeff * self._ins2_data_spline(time)
+        
+    def get_yt(self, time: np.ndarray):
+        """
+        Calculates the vertical distance y (in m) at traveled by a point
+        in the center of the high side of the trough. This distance  is a 
+        function of the accumulation rate A as y(t)=int(A(ins(t)) dt) or 
```

Comment:
```suggestion
        function of the accumulation rate A as y(t)=integral(A(ins(t)) dt) or 
```

The function `int()` is misleading, because in most languages it means "cast to an integer". I think saying 'integral' makes it explicit.

Reply:
Done